testing/sdk_security.py

```diff
@@ -0,0 +1,201 @@
+import shakedown
+import requests
+import sdk_utils
+
+
+def grant(creds, user, acl, description, action="create"):
+    dcosurl = creds['dcosurl']
+    headers = creds['headers']
+
+    sdk_utils.out('Granting permission to {user} for {acl}/{action} ({description})'.format(
+        user=user, acl=acl, action=action, description=description))
+
+    # Create the ACL
+    create_endpoint = '{dcosurl}/acs/api/v1/acls/{acl}'.format(dcosurl=dcosurl, acl=acl)
+    r = requests.put(create_endpoint, headers=headers, json={'description': description}, verify=False)
+    if r.status_code != 201 and r.status_code != 409: # 201=created, 409=already exists
+        raise Exception('{} failed {}: {}'.format(create_endpoint, r.status_code, r.text))
+
+    # Assign the user to the ACL
+    assign_endpoint = '{dcosurl}/acs/api/v1/acls/{acl}/users/{user}/{action}'.format(
+        dcosurl=dcosurl, acl=acl, user=user, action=action)
+    r = requests.put(assign_endpoint, headers=headers, verify=False)
+    if r.status_code != 204 and r.status_code != 409: # 204=success, 409=already exists
+        raise Exception('{} failed {}: {}'.format(create_endpoint, r.status_code, r.text))
+
+
+def revoke(creds, user, acl, description, action="create"):
+    dcosurl = creds['dcosurl']
+    headers = creds['headers']
+    sdk_utils.out("Want to delete {user}+{acl}".format(user=user, acl=acl))
+
+
+def get_dcos_credentials():
+    dcosurl, err, rc = shakedown.run_dcos_command('config show core.dcos_url')
+    token, err, rc = shakedown.run_dcos_command('config show core.dcos_acs_token')
+    headers = {
+        'Content-Type': 'application/json',
+        'Authorization': 'token={}'.format(token.strip()),
+    }
+    creds = { 'dcosurl': dcosurl.strip(), 'headers': headers }
```

Comment:
Could we perhaps just do `return dcosurl.strip(), headers`?

When the variables are actually used later, they are split up anyway:
```
dcosurl = creds['dcosurl']
headers = creds['headers']
```
Removing that dict would just be one less data structure to mess with. 

Also, I think we should move towards adding type hints for our method arguments in which case the choice is between:
`(creds: typing.Dict[str, typing.Union[str, typing.Dict[str, str]])` vs `(dcosurl: str, headers: dict)` so the latter would be preferable